flink-connectors/flink-sql-connector-elasticsearch6/pom.xml

```diff
@@ -135,6 +135,10 @@ under the License.
 									<pattern>com.fasterxml.jackson</pattern>
 									<shadedPattern>org.apache.flink.elasticsearch6.shaded.com.fasterxml.jackson</shadedPattern>
 								</relocation>
+								<relocation>
+									<pattern>com.github.mustachejava</pattern>
+									<shadedPattern>org.apache.flink.elasticsearch6.shaded.com.github.mustachejava</shadedPattern>
+								</relocation>
```

Comment:
Sorry for asking again but why the relocation?

Reply:
Same here, bumping elastic brought in this dependency, to keep the jar consistent I relocated it like the other ones